### PR TITLE
Nodemailer Mail sender

### PR DESCRIPTION
**Pull request by dhairyagothi**
Closes: #494  

- **Title** :  Nodemailer
- **Name:** :  Dhairya gothi
- **Idenitfy yourself:** :  GSSOC

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by dhairyagothi** [comment]
@Avdhesh-Varshney sir please merge PR

**Comment by Avdhesh-Varshney** [comment]
Properly follow the instructions. I'm not told to anyone again & again.

**Comment by dhairyagothi** [comment]
Done @Avdhesh-Varshney

File: Node-JS-Projects/Intermediate/Gmail-Nodemailer/readme.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+**Step 4:** Go to `http://localhost:8001` and enter url of the webpage you want to access contents of
+
+## Voila! Have fun using the site.
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+
+- Web page user interface
+ ![alt text](image.png)
```
Comment (Avdhesh-Varshney) [inline Node-JS-Projects/Intermediate/Gmail-Nodemailer/readme.md:67]: review it

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by dhairyagothi** [comment]
@Avdhesh-Varshney please tell anything else i need to change ?

File: Node-JS-Projects/Intermediate/Gmail-Nodemailer/readme.md (hunk 1 above)
Comment (dhairyagothi) [reply Node-JS-Projects/Intermediate/Gmail-Nodemailer/readme.md]: resolved

**Comment by dhairyagothi** [comment]
@Avdhesh-Varshney please merge PR 31 july is last date for contributing PR

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Enlist your project in readme file

**Comment by dhairyagothi** [comment]
done @Avdhesh-Varshney

**Review by Avdhesh-Varshney — APPROVED** [review]
Please fill this survey form https://forms.gle/Wvsp9N8TDF1WPStq9

**Pull request merged**